pkg/trace/api/debugger.go

```diff
@@ -90,9 +104,9 @@ func getDirector(hostTags string, cidProvider IDProvider, containerTags func(str
 			tags = fmt.Sprintf("%s,%s", tags, qtags)
 		}
 		maxLen := len(tags)
-		if maxLen > logsIntakeMaximumTagsLength {
-			log.Warn("Truncating tags in debugger endpoint. Got %d, max is %d.", maxLen, logsIntakeMaximumTagsLength)
-			maxLen = logsIntakeMaximumTagsLength
+		if maxLen > intakeMaximumTagsLength {
+			log.Warn("Truncating tags in debugger endpoint. Got %d, max is %d.", maxLen, intakeMaximumTagsLength)
```

Comment:
should this still say "debugger endpoint", or should it specificy the URL here?

Reply:
:+1: I've added the path to the log message, this way we can disambiguate between the two endpoints. I'm not printing the entire URL to prevent printing all the tags in the log. 

I've also fixed the log method - should have been `Warnf` not `Warn` here.